### ISSUE TITLE
build failing on openbsd/armv5

### ISSUE THREAD

**Issue by stuartnelson3** [prometheus/alertmanager#723]
~~this commit broke the dragonfly build:~~

https://github.com/golang/sys/commit/9a7256cb28ed514b4e1e5f68959914c4c28a92e0

~~check it out and see how to fix it~~

Currently broken on openbsd/armv5

**Comment by stuartnelson3** [comment]
https://github.com/golang/go/issues/20055

**Comment by stuartnelson3** [comment]
The process to contribute to the go repo is ... prohibitive. I committed the fix directly here and am waiting for someone to do https://github.com/golang/go/issues/20055.

https://github.com/prometheus/alertmanager/commit/e9cb88e15a3b28cb6c0afa3f20ef34a1edb12140

**Comment by stuartnelson3** [comment]
Now it's failing on openbsd-armv5:

```
# openbsd-armv5

>> building binaries

 >   alertmanager

 >   amtool

# github.com/prometheus/alertmanager/vendor/golang.org/x/sys/unix

/go/src/github.com/prometheus/alertmanager/vendor/golang.org/x/sys/unix/flock.go:18: undefined: Flock_t

/go/src/github.com/prometheus/alertmanager/vendor/golang.org/x/sys/unix/sockcmsg_unix.go:36: undefined: Cmsghdr

/go/src/github.com/prometheus/alertmanager/vendor/golang.org/x/sys/unix/sockcmsg_unix.go:42: undefined: Cmsghdr

!! command failed: build -o .build/openbsd-armv5/amtool -ldflags -X github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/version.Version=0.5.1 -X github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/version.Revision=e9cb88e15a3b28cb6c0afa3f20ef34a1edb12140 -X github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/version.Branch=master -X github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/version.BuildUser=root@d6492630dae2 -X github.com/prometheus/alertmanager/vendor/github.com/prometheus/common/version.BuildDate=20170421-14:39:39  -extldflags '-static' -a -tags netgo github.com/prometheus/alertmanager/cmd/amtool: exit status 2
```

**Comment by stuartnelson3** [comment]
Fixed by #732
